src/main/java/teammates/common/datatransfer/FeedbackContributionQuestionDetails.java

```diff
@@ -891,22 +892,23 @@ else if (i == Const.POINTS_NOT_SUBMITTED)
      * @return points in text form "Equal Share..." with html formatting for colors.
      */
     public static String convertToEqualShareFormatHtml(int i) {
-        if (i == Const.INT_UNINITIALIZED)
+        if (i == Const.INT_UNINITIALIZED) {
             return "<span class=\"color_neutral\">N/A</span>";
-        else if (i == Const.POINTS_NOT_SUBMITTED)
+        } else if (i == Const.POINTS_NOT_SUBMITTED) {
             return "<span class=\"color_neutral\"></span>";
-        else if (i == Const.POINTS_NOT_SURE)
+        } else if (i == Const.POINTS_NOT_SURE) {
             return "<span class=\"color-negative\">Not Sure</span>";
-        else if (i == 0)
+        } else if (i == 0) {
             return "<span class=\"color-negative\">0%</span>";
-        else if (i > 100)
+        } else if (i > 100) {
             return "<span class=\"color-positive\">Equal Share +" + (i - 100) + "%</span>";
-        else if (i < 100)
+        } else if (i < 100) {
             return "<span class=\"color-negative\">Equal Share -" + (100 - i) + "%</span>";
-        else if (i == 100)
+        } else if (i == 100) {
             return "<span class=\"color_neutral\">Equal Share</span>";
-        else
+        } else {
             return "";
+        }
```

Comment:
Actually, the last `else` is unnecessary, and the last condition `i == 100` can be the `else` clause, because if `i < 100` and `i > 100` are both violated, it must be `i == 100`.